### PR TITLE
Hy n migrate 71 712

### PR DESCRIPTION
**Pull request by JohannaOtt**
SVC-1480

### CODE REVIEW

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml [hunk 1]
```diff
@@ -392,31 +392,35 @@ Die lokale ID wird aus dem Namen der Ziel-Objektart, der AAA-ObjektID des Quell-
...
-			nilReason( 'other:unpopulated' )&#13;
-		}&#13;
-		spelling {&#13;
-			SpellingOfName {&#13;
-				text( regionalsprache )&#13;
-				script( 'Latn' )&#13;
+&#13;
+if (regionalsprache) {&#13;
+	regionalsprache.p.each { reg -&gt;&#13;
+		_target{&#13;
```
Comment (florianesser) [inline annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml:398]: Code style: `_target {` (multiple occurrences in file)

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml [hunk 2]
```diff
@@ -392,31 +392,35 @@ Die lokale ID wird aus dem Namen der Ziel-Objektart, der AAA-ObjektID des Quell-
...
+			GeographicalName {&#13;
+				sourceOfName{&#13;
```
Comment (florianesser) [inline annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml:400]: Code style: `sourceOfName {` (multiple occurrences in file)

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml [hunk 3]
```diff
@@ -392,31 +392,35 @@ Die lokale ID wird aus dem Namen der Ziel-Objektart, der AAA-ObjektID des Quell-
...
+				spelling {&#13;
+					SpellingOfName {&#13;
+						text( reg )&#13;
+						script( 'Latn' )&#13;
+					}&#13;
+				}&#13;
+				pronunciation {&#13;
+					nilReason( 'other:unpopulated' )&#13;
+				}&#13;
+				language( 'deu' )&#13;
```
Comment (florianesser) [inline annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml:412]: The change that `language` is now filled is not announced in the commit message.

**Review by florianesser — COMMENTED** [review]
Generally LGTM, mentioned a couple of minor issues in the comments.

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml (hunk 1 above)
Comment (JohannaOtt) [reply annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml]: Done

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml (hunk 2 above)
Comment (JohannaOtt) [reply annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml]: Done

File: annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml (hunk 3 above)
Comment (JohannaOtt) [reply annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml]: I decided to remove the language for GeographicalName objects resulting from "regionalsprache" as that is not German.

Comment (JohannaOtt) [reply annex-1/mappings7.1/Hydrography/aaa-hy-p.halex.alignment.xml]: Added some information on the language handling to the commit message.

**Comment by JohannaOtt** [comment]
@florianesser As no official review has been done, I can not request your re-review using the github functionality. Please re-review.

**Review by florianesser — APPROVED** [review]
LGTM

**Pull request merged**